src/controllers/mainController.ts

```diff
@@ -398,12 +398,13 @@ export default class MainController implements vscode.Disposable {
     /**
      * Manage connection profiles (create, edit, remove).
      */
-    private onManageProfiles(): Promise<boolean> {
+    private async onManageProfiles(): Promise<void> {
         if (this.canRunCommand()) {
             Telemetry.sendTelemetryEvent('ManageProfiles');
-            return this._connectionMgr.onManageProfiles();
+            await this._connectionMgr.onManageProfiles();
+            return this._objectExplorerProvider.refresh(undefined);
         }
```

Comment:
This can be removed as well. Since this function has a void return type the return would typically be on the next line after the call to refresh, but in this case it's not even necessary since there's nothing else that's going to execute so no need to early return.

Reply:
refresh returns a void Promise, and that's what I return here

Reply:
No it doesn't - it returns just a void
https://github.com/microsoft/vscode-mssql/blob/master/src/objectExplorer/objectExplorerProvider.ts#L26

(the async handles wrapping it in the Promise for you)

While that's still technically fine the practice should be if your return type is void you only ever use `return;` It just makes the intent clearer and reduces chance of confusion.

Reply:
That's what I meant with the async, since it's wrapped in a promise, returning on the same line should be fine, but sure I'll add the return on the next line if you want.

Reply:
Thanks. It's definitely a nit but we should really be consistent with return usage in all our code and this is the style we general use elsewhere.